pages/people.md

```diff
@@ -126,6 +126,7 @@ permalink: /people/
 |             | Vincent Russell     |                                                                   |                       |
 |             | Volth               |                                                                   |                       |
 |             | Wil Selwood         | [Satellite Applications Catapult][SACAT]                          | [GMT][GMT]/[BST][BST] |
+| glitch      |                     |                                                                   |                       |
```

Comment:
Sorry @glitch , that field is for Apache user names, not GitHub user names. :)
If you prefer not to be named, we could put your name as "GitHub user: glitch".